mtags/src/main/scala-2/scala/meta/internal/metals/Trees.scala

```diff
@@ -67,7 +67,9 @@ final class Trees {
     Option(PathIO.extension(Paths.get(fileUri))).collect {
       case "scala" => dialects.Scala
       case "sbt" => dialects.Sbt
-      case "sc" => dialects.Sbt
+      case "sc" =>
+        dialects.Scala213
```

Comment:
We should still use `dialects.Sbt` for worksheets running in 2.12.x Scala version. You can use `
  BuildInfo.scalaCompilerVersion` to check

Reply:
I think it's OK to use the 2.13 dialect for all worksheets. The users will still get syntax errors when they save a file that uses 2.13-only syntax with a 2.12 classpath

Reply:
This dialect only affects the syntax errors that we display on keystroke. It's helpful for setting errors like "unbalanced parentheses" or "missing closing quote". It's not latency sensitive that the user gets a syntax errors when they for example use literal types (2.13-only syntax)

Reply:
Yeah, you're actually right after thinking about it for a while.